core-services/template-deprecation/_allowlist.yaml

```diff
@@ -1267,6 +1267,9 @@ templates:
         canary-release-openshift-origin-installer-e2e-aws-4.8-cnv:
           generated: false
           kind: unknown
+        pull-ci-kubevirt-kubevirt-tekton-tasks-main-e2e-tests-cluster-scope:
+          generated: true
+          kind: presubmit
```

Comment:
Huh I'm surprised to see this here, was there an iteration of this PR that added a template job? Because now it isn't, and this should not be necessary.

Reply:
it probably got here due to one of my rebases from earlier generation

Reply:
removed